lighthouse-cli/cli-flags.js

```diff
@@ -152,6 +161,17 @@ function getFlags(manualArgv) {
       .default('enable-error-reporting', undefined) // Undefined so prompted by default
       .default('channel', 'cli')
       .check(/** @param {LH.CliFlags} argv */ (argv) => {
+        // ".middleware" does not exist in this version of yargs, so do some preprocessing here.
+        for (const _key of Object.keys(argv)) {
```

Comment:
I don't love that this is here and I'm not sure we can actually do this across the board.

(for example `--chrome-flags="--window-size 800,600"`)

WDYT about just doing this to our specific `.array` properties minus `blockedUrlPatterns`?

Reply:
ah ok. sounds like either way we need a hard coded list (either a blacklist of a whitelist). Whitelist is easier to reason about so let's do that :)